src/main/scala/kinesis/mock/KinesisMockService.scala

```diff
@@ -24,6 +25,13 @@ object KinesisMockService extends IOApp {
           "Logging Cache Config"
         )
         cache <- Cache(cacheConfig)
+        _ <- cacheConfig.initializeStreams.toList.flatten.traverse(s =>
+          logger
+            .info(
+              s"Initializing stream '${s.streamName}' (shardCount=${s.shardCount})"
+            )
+            .flatTap(_ => cache.createStream(s, context))
```

Comment:
We may want to call a `sleep` that leverages the createStreamDuration configuration, so that the streams are up when the service is up.